src/bidiMapper/domains/script/RealmStorage.ts

```diff
@@ -62,13 +63,21 @@ export class RealmStorage {
       }
       if (
         filter.browsingContextId !== undefined &&
-        filter.browsingContextId !== realm.browsingContextId
+        (!(realm instanceof WindowRealm) ||
+          filter.browsingContextId !== realm.browsingContext.id)
       ) {
         return false;
       }
       if (
         filter.navigableId !== undefined &&
-        filter.navigableId !== realm.navigableId
+        (!(realm instanceof WindowRealm) ||
+          filter.navigableId !== realm.browsingContext.navigableId)
+      ) {
+        return false;
+      }
+      if (
+        filter.sandbox !== undefined &&
+        (!(realm instanceof WindowRealm) || filter.sandbox !== realm.sandbox)
```

Comment:
`filter.sandbox` is used in [`BrowsingContextImpl`](https://github.com/GoogleChromeLabs/chromium-bidi/blob/477d1f28c39f02777a4801acdcaea09311f1fdc1/src/bidiMapper/domains/context/BrowsingContextImpl.ts#L272), and the `sandbox` filtering logic can be removed there.

Reply:
Not sure what you mean here. Why can the sandbox logic be removed?

Reply:
sgtm

Reply:
Sorry, I reverted it because we use the filter for the default realm.

Reply:
The default realm logic can be moved to BrowsingContext. But this is optional.